docs/release-notes/8.4.4-4.md

```diff
@@ -0,0 +1,71 @@
+
+### MySQL 8.4.4
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.4.4 and included in Percona Server for MySQL are the following:
+
+* Fixed an assertion in debug builds where certain IO buffer serializations caused system hangs. (Bug #37139618)
+
+* Resolved a failure when dropping the primary key and adding a new `AUTO_INCREMENT` column as the primary key in descending order using the `INPLACE` algorithm resulted in failure. (Bug #36658450)
+
+* Fixed incorrect results, including missing rows, in queries that used a descending primary key with the `index_merge` optimization. (Bug #106207, Bug #33767814)
+
+* Addressed a replication channel issue where MySQL failed to stop the channel properly when large transactions were being processed, and `STOP REPLICA` was requested. This issue also prevented graceful server shutdown, requiring process termination or system restart. (Bug #115966, Bug #37008345)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.4.4 release notes](https://dev.mysql.com/doc/relnotes/mysql/8.4/en/news-8-4-4.html).
+
+## Improvements
+
+[PXC-4469](https://perconadev.atlassian.net/browse/PXC-4469): Implements the [Clone plugin for State Snapshot Transfer (SST) Method](https://docs.percona.com/percona-xtradb-cluster/8.4/clone-sst.html).
+
+## Bug fixes
+
+* [PXC-4173](https://perconadev.atlassian.net/browse/PXC-4173): The Percona XtraDB Cluster node stalled when parallel replication workers executed Data Definition Language (DDL) through an asynchronous node.
+
+* [PXC-4390](https://perconadev.atlassian.net/browse/PXC-4390): The `wsrep_group_commit_queue` option was removed because caused deadlock bugs.
+
+* [PXC-4498](https://perconadev.atlassian.net/browse/PXC-4498): The Percona XtraDB Cluster node stalled and caused the cluster to stop responding.
+
+* [PXC-4393](https://perconadev.atlassian.net/browse/PXC-4393): On the donor side, if the pipeline had more than two commands, only the last command’s failure was detected, but the error was shown for the second-to-last command.
+
+* [PXC-4404](https://perconadev.atlassian.net/browse/PXC-4404): Setting the `wsrep_preordered=ON` option caused protocol violations.
+
+* [PXC-4504](https://perconadev.atlassian.net/browse/PXC-4504): The `DROP PROCEDURE/FUNCTION IF EXISTS` option generated a local Global Transaction Identifier (GTID) event.
+
+* [PXC-4506](https://perconadev.atlassian.net/browse/PXC-4506): The asynchronous replication threads on a Percona XtraDB Cluster node hung when resuming from an applier error after it was fixed.
+
+* [PXC-4509](https://perconadev.atlassian.net/browse/PXC-4509): The asynchronous replication hung on the Percona XtraDB Cluster node if the replica received Global Transaction Identifier (GTID) from the source that had already been committed on it.
```

Comment:
Please remove. This one was created by PXC-4173

Reply:
removed